staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -869,6 +869,10 @@ func (cfgCtlr *configController) maybeReap(plName string) {
 		klog.V(7).Infof("plName=%s, plState==nil", plName)
 		return
 	}
+	if plState.queues == nil {
+		klog.V(7).Infof("plName=%s, plState.quiescing=%v, plState.numPending=%d, plState.queues==nil", plName, plState.quiescing, plState.numPending)
```

Comment:
This log message should explicitly say that the priority level is exempt.  BTW, for an exempt priority level it is certain that: it is not quiescing and it has no queues; there is no value in printing those things in a message that also explicitly says "exempt".  The `numPending` is not fixed, but also not relevant, so it would be fine to not print that either.